_html/scss/modules/_user-form.scss

```diff
@@ -0,0 +1,35 @@
+.user-form {
+    margin-top: 10rem;
+
+    .form-item-wrapper-content {
+        display: inline-block;
+        width: calc(50% - 1.2rem);
+        vertical-align: middle;
+
+        &.form-item-pass,
+        .form-item-field-gender label {
+            width: 100%;
+        }
+    }
+
+    .form-item-wrapper {
+        margin-bottom: 2rem;
+
+        &>:first-child {
```

Comment:
You can remove the > . Because using &:first-child will target the first direct child of the parent no matter what. This applies to the selector on line 22, so please remove both > .